inference_perf/reportgen/base.py

```diff
@@ -16,8 +16,17 @@
 from typing import Tuple
 
 
-class Metric(BaseModel):
-    name: str
+class MetricsSummary(BaseModel):
+    total_requests: int
+    avg_prompt_tokens: float
+    avg_completion_tokens: float
+    avg_latency: float
```

Comment:
This is a good start. Let's plan to measure the ones we called out in the spec - https://github.com/kubernetes-sigs/inference-perf/blob/main/docs/design.md#metrics-to-collect. We also need to report different distributions with mean being one - median, p90, p99 would also be needed. But we can do it in a follow up change.